.github/release-drafter.yml

```diff
@@ -0,0 +1,18 @@
+# Inherit base settings from https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
+_extends: .github
+name-template: REST List Parameter v$RESOLVED_VERSION
```

Comment:
This should probably have a different name. The tag-template as well

Reply:
All I can say in my defense ... :facepalm:  I am sorry for this f up.

Reply:
It's all good. If I got a nickel for every time I made a copy paste mistake I'd be retired by now 😉